packages/tools/src/tools/annotation/planarFreehandROITool/interpolation/interpolateSegmentPoints.ts

```diff
@@ -0,0 +1,84 @@
+import { Types } from '@cornerstonejs/core';
+import { interpolatePoints } from './algorithms/bspline';
+
+/**
+ * Returns a list of uniform distributed values. This list contains the max amount of values which has at least a minimum distance between two consecutive values.
+ * minDistributionDistance means the min distance between two consecutive distributed values.
+ * Closed interval contains the min/max values.
+ *
+ * Formula for reference
+ * For given {x ∈ R | x ≥ 0} and {minDis ∈ R | minDis ≥ 0}, ∃ D(x) where D(x) ≥ a and D(x) ≤ b  =>
+ *         |
+ * D(x)  = |                  (b - a)
+ *         |  round( ------------------------ * x  )   + a
+ *         |                (b - a + 1)
+ *         |        round( -----------  )
+ *         |                 minDis
+ */
+function getContinuosUniformDistributionValues(
```

Comment:
Minor quibble - getContinuousUniformDistributionValues
There is a second u in Continuous